DigitalOffice/Tests/TimeManagementServiceTests/Commands/CreateWorkTimeCommandTests.cs

```diff
@@ -0,0 +1,101 @@
+
+        private ICreateWorkTimeCommand command;
+        private CreateWorkTimeRequest request;
+        private DbWorkTime createdWorkTime;
+
+        [SetUp]
+        public void SetUp()
+        {
+            validatorMock = new Mock<IValidator<CreateWorkTimeRequest>>();
+            mapperMock = new Mock<IMapper<CreateWorkTimeRequest, DbWorkTime>>();
+            repositoryMock = new Mock<IWorkTimeRepository>();
+
+            command = new CreateWorkTimeCommand(validatorMock.Object, mapperMock.Object, repositoryMock.Object);
+
+            request = new CreateWorkTimeRequest()
+            {
+                ProjectId = Guid.NewGuid(),
+                Date = new DateTime(2020, 7, 29, 0, 30, 0),
+                Time = new TimeSpan(8, 30, 0),
+                Title = "I was working on a very important task",
+                Description = "I was asleep. I love sleep. I hope I get paid for this.",
+                WorkerUserId = Guid.NewGuid()
+            };
+
+            createdWorkTime = new DbWorkTime()
+            {
+                Id = Guid.NewGuid(),
+                ProjectId = Guid.NewGuid(),
+                Date = new DateTime(2020, 7, 29, 0, 30, 0),
+                Time = new TimeSpan(8, 30, 0),
+                Title = "I was working on a very important task",
+                Description = "I was asleep. I love sleep. I hope I get paid for this.",
+                WorkerUserId = Guid.NewGuid()
+            };
+        }
+
+        [Test]
+        public void FailCreateNewWorkTimeAccordingToValidator()
+        {
+            validatorMock
+                .Setup(x => x.Validate(It.IsAny<IValidationContext>()).IsValid)
+                .Returns(false);
+
+            Assert.Throws<ValidationException>(() => command.Execute(request));
+        }
+
+        [Test]
+        public void FailCreateNewWorkTimeAccordingToRepository()
+        {
+            validatorMock
+                 .Setup(x => x.Validate(It.IsAny<IValidationContext>()).IsValid)
+                 .Returns(true);
+
+            mapperMock
+                .Setup(X => X.Map(It.IsAny<CreateWorkTimeRequest>()))
+                .Returns(createdWorkTime);
+
+            repositoryMock
+                .Setup(x => x.CreateWorkTime(It.IsAny<DbWorkTime>()))
+                .Throws(new Exception());
+
+            Assert.Throws<Exception>(() => command.Execute(request));
+        }
+
+        [Test]
+        public void SuccesfulCreateNewWorkTimer()
+        {
+            validatorMock
+                 .Setup(x => x.Validate(It.IsAny<IValidationContext>()).IsValid)
+                 .Returns(true);
+
+            mapperMock
+                .Setup(X => X.Map(It.IsAny<CreateWorkTimeRequest>()))
+                .Returns(createdWorkTime);
+
+            repositoryMock
+                .Setup(x => x.CreateWorkTime(It.IsAny<DbWorkTime>()))
+                .Returns(createdWorkTime.Id);
+
+            Assert.AreEqual(createdWorkTime.Id, command.Execute(request));
```

Comment:
Fixed.

Reply:
Why not to verify all the subsequent mocks?

Reply:
Like that: 
![image](https://user-images.githubusercontent.com/61093487/89462967-dea9bc80-d776-11ea-877e-e004c10f2b0c.png)

Reply:
One note: I'm not sure I'm verifying validators right.
It throws me an exception when I write <...> Validate(request), so instead I wrote in the verification that validator validates any IValidationContext supported objects types

Reply:
Ah, I see. You verified repo like i did the validator. Alright, will use this in future tests as well (instead of passing actual arguments).
Or actual arguments are also good?